static/src/stylesheets/module/content-garnett/_types.scss

```diff
@@ -106,15 +233,92 @@ $quote-tail: 25px; // ********** Article Type **********
             &:focus,
             &:hover {
                 .inline-tone-fill {
-                    fill: $neutral-garnett-1;
+                    fill: $neutral-gt-1;
                 }
 
                 .contact {
-                    color: $neutral-garnett-1;
+                    color: $neutral-gt-1;
                 }
             }
         }
     }
+    // *************** Header and Body Styles ***************
+    .gs-container {
+        @include mq(tablet, desktop) {
+            padding-top: $gs-baseline;
+        }
+    }
+
+    .content__article-body,
+    .immersive-main-media__headline-container .content__main-column {
+        @include mq(desktop) {
+            border-left: solid 1px $neutral-5;
+            padding-left: $gs-gutter/2;
+            margin-left: 0;
+            padding-top: $gs-gutter/2;
+            padding-bottom: 100px;
+        }
+        @include mq(leftCol) {
+            margin-left: -$gs-gutter/2;
+        }
+    }
+
+    .content__secondary-column {
+        padding-top: $gs-gutter/2;
+    }
+
+    .mobile-only {
+        display: block!important;
+    }
+
+    .caption--main.caption--img {
+        position: absolute;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        background: rgba(22, 22, 22, .8);
+        color: #ffffff;
+        display: none;
+        padding: .375rem 2.5rem .75rem .625rem;
```

Comment:
go with pixels here – they'll be converted to rems in build, and easier to reason about!